### PR TITLE
Part 2

### PR DESCRIPTION
**Pull request by samyou-softwire**

### CODE REVIEW

File: src/rovers.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import {Welcome} from "./server";
+import { config } from "dotenv"
+config();
+const API_KEY = <string>process.env.API_KEY;
+
+export function addRoverEndpoint(router: Router) {
+    router.get('/rovers', async (req, res) => {
+        console.log(API_KEY);
+
+        const response: AxiosResponse<Welcome, any> = await axios.get(
```
Comment (kwunkiuma) [inline src/rovers.ts]: I think we can simplify this just a bit using `axios.get<Welcome>` 🙂

File: src/server.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import express from "express";
+import { config } from "dotenv"
+config();
```
Comment (kwunkiuma) [inline src/server.ts]: I'm not sure we need this in this file?

**Review by kwunkiuma — CHANGES_REQUESTED** [review]
Looking good! Just some very minor nitpicks 😄

File: src/server.ts (hunk 2 above)
Comment (samyou-softwire) [reply src/server.ts]: nope 👍

File: src/rovers.ts (hunk 1 above)
Comment (samyou-softwire) [reply src/rovers.ts]: yep 👍

**Review by kwunkiuma — APPROVED** [review]
LGTM, thanks!

**Pull request merged**